libs/blocks/how-to/how-to.js

```diff
@@ -47,7 +47,19 @@ const setJsonLd = (heading, description, mainImage, stepsLd) => {
 };
 
 const getImage = (el) => el.querySelector('picture') || el.querySelector('a[href$=".svg"');
-const getVideo = (el) => el.querySelector('a[href*=".mp4"]');
```

Comment:
You can keep this if you adapt the intersection observer for the videos
```suggestion
const getVideo = (el) => el.querySelector('video');
```